portfolio/src/main/java/com/google/sps/utils/CommentDataStore.java

```diff
@@ -41,4 +47,16 @@ public void store(Comment comment) {
         }
         return comments.build();
     }
+
+    /**
+     * Deletes a specific comment in Datastore.
+     * @param kind Comment entity kind name.
+     * @param id Target comment entity id.
+     */
+    @Override
+    public void delete(String kind, long id) {
```

Comment:
Why do we receive kind if we already have KIND defined above?

Reply:
Fixed